APIReference.md

```diff
@@ -149,28 +147,12 @@ return a promise.</p></dd>
 ***Deprecated***
 
 **Kind**: instance method of [<code>Collection</code>](#Collection)  
-<a name="Collection+createIndex"></a>
-
-### collection.createIndex(index, options) ⇒
-**Kind**: instance method of [<code>Collection</code>](#Collection)  
-**Returns**: <p>any</p>  
-
-| Param |
-| --- |
-| index | 
-| options | 
+<a name="Collection+count"></a>
 
-<a name="Collection+dropIndexes"></a>
+### ~~collection.count()~~
+***Deprecated***
```

Comment:
I really wish we could remove this... But looks like Mongoose still has `count()`.

Reply:
Got it, ll leave it for now.